README.md

```diff
@@ -25,6 +25,16 @@ $ cargo install thokr
 $ docker run -it coloradocolby/thokr
 ```
 
+## Arch Linux
+
+On Arch Linux, you can install it from AUR:
+
+``` bash
+paru -S thokr-git
```

Comment:
Ideally I'd like if the recommended way to download somehow didn't assume the user is using paru, yay, etc. Let me think about how to do this 🤔 I'll see if I can find examples in other repos I like!

Reply:
Ok no problem, I have put paru because I think it is the one most people use right now (including me).